app/src/androidTest/java/com/github/palFinderTeam/palfinder/UIMockMeetUpRepositoryModule.kt

```diff
@@ -50,6 +51,7 @@ object UIMockMeetUpRepositoryModule {
             val key = counter.toString()
             db[key] = newMeetUp.copy(uuid = key)
             counter += 1
+            Log.d("chat","chalut " + db[key]!!.participantsId)
```

Comment:
don't forget to delete it if it is for local test purpose

Reply:
oh yes I delete it thank you